pkg/covermerger/covermerger.go

```diff
@@ -223,18 +223,19 @@ type FileMergeResult struct {
 	*MergeResult
 }
 
-func MergeCSVData(config *Config, reader io.Reader, results chan<- *FileMergeResult) error {
+func MergeCSVData(c context.Context, config *Config, reader io.Reader, results chan<- *FileMergeResult) error {
 	var schema []string
 	csvReader := csv.NewReader(reader)
 	if fields, err := csvReader.Read(); err != nil {
 		return fmt.Errorf("failed to read schema: %w", err)
 	} else {
 		schema = fields
 	}
-	errStreamChan := make(chan error, 1)
+	errStreamChan := make(chan error, 2)
```

Comment:
Nit: wouldn't it be cleaner to use an errgroup here as well?

Reply:
I also think so...